mesonbuild/scripts/externalproject.py

```diff
@@ -41,9 +41,7 @@ def write_stampfile(self) -> None:
 
     def supports_jobs_flag(self) -> bool:
         p, o, e = Popen_safe(self.make + ['--version'])
-        if p.returncode == 0 and ('GNU Make' in o or 'waf' in o):
-            return True
-        return False
+        return bool(p.returncode == 0 and ('GNU Make' in o or 'waf' in o))
```

Comment:
Do we really need `bool()` here? it looks like that would simplify to `bool and (bool or bool)`.